src/database.ts

```diff
@@ -1511,36 +1739,40 @@ class Database extends ServiceObject {
     optionsOrRunFn: RunTransactionOptions | RunTransactionCallback,
     fn?: RunTransactionCallback
   ): void {
-    let options: RunTransactionOptions;
-    let runFn: RunTransactionCallback;
-
-    if (is.fn(optionsOrRunFn)) {
-      runFn = optionsOrRunFn as RunTransactionCallback;
-      options = {};
-    } else {
-      options = optionsOrRunFn as RunTransactionOptions;
-      runFn = fn!;
-    }
+    const runFn =
+      typeof optionsOrRunFn === 'function'
+        ? (optionsOrRunFn as RunTransactionCallback)
+        : fn!;
```

Comment:
```suggestion
        : fn;
```
as `fn` is stated optional